cookbooks/bach_spark/recipes/spark_package.rb

```diff
@@ -0,0 +1,36 @@
+spark_tar_file = node[:spark][:download][:file]
+spark_tar_url = node[:spark][:download][:url]
+spark_download_dir = node[:spark][:download][:dir]
+spark_install_dir = node[:spark][:package][:base]
+spark_pkg_prefix = node[:spark][:package][:prefix]
+spark_pkg_version = node[:spark][:package][:version]
+
+gem_package "fpm" do
+   action :nothing
+end.run_action(:install)
```

Comment:
You should not need to `run_action` here since Chef is not needing to load the Gem, this will install before the `bash` action [below](https://github.com/bloomberg/chef-bach/pull/141/files#diff-4f690f0cb3248f506a62d866e231e0eeR17)?